src/lv_misc/lv_color.h

```diff
@@ -305,9 +305,14 @@ static inline uint32_t lv_color_to32(lv_color_t color)
 #elif LV_COLOR_DEPTH == 16
 #if LV_COLOR_16_SWAP == 0
     lv_color32_t ret;
-    ret.ch.red   = color.ch.red * 8;   /*(2^8 - 1)/(2^5 - 1) = 255/31 = 8*/
-    ret.ch.green = color.ch.green * 4; /*(2^8 - 1)/(2^6 - 1) = 255/63 = 4*/
-    ret.ch.blue  = color.ch.blue * 8;  /*(2^8 - 1)/(2^5 - 1) = 255/31 = 8*/
+    /**
+     * Per https://docs.google.com/spreadsheets/d/1PppX8FJpddauAPasHwlNgIPGIuPGPNvRbhilIQ8w-7g/edit#gid=0
```

Comment:
Not a big deal, but this spreadsheet doesn't seem to be publicly viewable.

I suppose that either that should be changed or this comment should be removed.

Reply:
Will move abbreviated logic in to code comment...

Reply:
BTW, I thought I had made the spreadsheet public, but I guess it didn't take.
I made it public and confirmed it is.
I'll still abbreviate the logic in a code comment...